config/services.php

```diff
@@ -149,4 +151,9 @@
                 service(InternalGrpcWorker::class),
             ]);
     }
+
+    if (class_exists(WorkflowInterface::class)) {
+        $services->set('temporal.exception_interceptor', ExceptionInterceptor::class)
```

Comment:
shouldn't `ExceptionInterceptorInterface` be used instead?

```suggestion
        $services->set('temporal.exception_interceptor', ExceptionInterceptorInterface::class)
```